planetary-social/nos
# #1601: Go to Your Feed tip

## Issues covered
#1601 

## Description
Shows a "Go to your feed..." tip at the bottom of the Discover tab. It doesn't match the design, but getting the popover to work was impossible in the time I had with the knowledge I had. Maybe we can still make it happen.

~- [ ] try to use a real popover instead of this fake one~ Linda is fine with this approach
- [x] show the tip after the user has followed 3 accounts
- [x] don't show the tip to people who have been using the app forever
- [x] dismiss the tip when the user taps the Feed tab

## How to test
1. Navigate to Discover
2. Observe the tip at the bottom
3. If you dismiss it and want to test again, you can delete the app and reinstall. Or, for developer testing, add this line to `NosApp.init()` after the `Tips.configure()` line:

```
Tips.showAllTipsForTesting()
```

## Screenshots/Video

| Before | After |
|--------|--------|
| ![Simulator Screenshot - iPhone 16 Pro - 2024-10-11 at 17 22 39](https://github.com/user-attachments/assets/455b683d-10ea-4848-ab3f-b20bd7c71cad) | ![Simulator Screenshot - iPhone 16 Pro - 2024-10-11 at 17 22 36](https://github.com/user-attachments/assets/3ffcf519-1378-4a93-9407-f19543385a1e) |
